bootloader/src/pyi_pythonlib.c

```diff
@@ -280,18 +293,6 @@ pyi_wargv_from_argv(int argc, char ** argv)
     return wargv;
 }
 
-void
-pyi_free_wargv(wchar_t ** wargv)
-{
-    wchar_t ** arg = wargv;
-
-    while (arg[0]) {
-        free(arg[0]);
-        arg++;
-    }
-    free(wargv);
-}
```

Comment:
Thanks for the explanation. This is reasonable and adding a sole prototype would be somewhat "fishy", either,